README.md

```diff
@@ -36,30 +36,42 @@ There are quite extensive requirements in annotation etc. data required for the
+| sample_list                       | empty                                      | Semicolon-separated list of sample names in the same order as `bamfile_list`                                                                                  |
+| possibleTumorSampleNamePrefixes   | "( tumor )"                                | Bash-array of tumor sample name prefixes.                                                                                                                     |
+| possibleControlSampleNamePrefixes | "( control )"                              | Bash-array of control sample name prefixes.                                                                                                                   |
+| REFERENCE_GENOME                  | empty                                      |                                                                                                                                                               |
+| CHR_SUFFIX                        | ""                                         | Suffix added to the chromosome names                                                                                                                          |
+| CHR_PREFIX                        | ""                                         | Prefix added to the chromosome names                                                                                                                          |
+| extractSamplesFromOutputFiles     | true                                       |                                                                                                                                                               |
+| CHROMOSOME_INDICES                | empty                                      | Bash-array of chromosome names to which the analysis should be restricted                                                                                     |
+| SEQUENCE_TYPE                     | WGS                                        | "WES" or "WGS"                                                                                                                                                |
+| EXOME_CAPTURE_KIT_BEDFILE         | see `analysisIndelCalling.xml`              | If `runTinda=true` and `SEQUENCE_TYPE=WES` you need a BED file with target regions.                                                                           |
+| VCF_NORMAL_HEADER_COL             | basename of control BAM, w/o ".bam" suffix | If `runTinda=true`. Can be a `\|`-separated list of identifiers. Note that the double quotes are need. E.g. `"buffy_coat1"`.                                                                                                             | 
```

Comment:
Since we have removed the pattern matching for the column names in the `checkSampleSwap_TiN.pl` and `VCF_NORMAL_HEADER_COL` and `VCF_TUMOR_HEADER_COL` are directly inferred from the BAM files now, we could remove them from the XML files and also here in the README.

Reply:
Yes this is related to the changes of this branch, but the respective code in `checkSampleSwap_TiN.sh` is not removed.

So you suggest remove the variables completely, then remove extraction of the column names from the BAM content, and the `if [[ -z $VCF_NORMAL_HEADER_COL ]]` in `checkSampleSwap_TiN.sh`. Instead we'd just have the following there:

```bash
VCF_TUMOR_HEADER_COL=`basename ${FILENAME_TUMOR_BAM} | sed 's/.bam$//'`
VCF_NORMAL_HEADER_COL=`basename ${FILENAME_CONTROL_BAM} | sed 's/.bam$//'`
```

?

Reply:
I added these changes to the PR.